ams/server/src/main/java/com/netease/arctic/server/ArcticManagementConf.java

```diff
@@ -103,6 +103,18 @@ public class ArcticManagementConf {
           .defaultValue(60000L)
           .withDescription("Interval for refreshing table metadata.");
 
+  public static final ConfigOption<Integer> EXTERNAL_CATALOGS_EXPLORER_THREAD_COUNT =
+          ConfigOptions.key("external-catalog-explorer.thread-count")
```

Comment:
AMS already supports the `refresh-external-catalogs.interval` configuration, can we expand the configuration to make these configurations more understandable to users, like :
```
refresh-external-catalogs:
    interval: 180000 # 3min
    thread-count: 10
    queue-size: 1000000
```

Reply:
Good point.